anyway/infographics_utils.py

```diff
@@ -1118,90 +1121,64 @@ def localize_items(request_params: RequestParams, items: Dict) -> Dict:
 
 
 @register
-class InjuredAccidentsWithPedestriansWidget(SubUrbanWidget):
+class InjuredAccidentsWithPedestriansWidget(UrbanWidget):
     name: str = "injured_accidents_with_pedestrians"
 
+    def validate_parameters(self):
+        # TODO: validate each parameter and display message accordingly
+        return self.request_params.location_info.get('yishuv_name') is not None and \
+            self.request_params.news_flash_obj.street1_hebrew is not None and  \
+            self.request_params.years_ago is not None
+
+    def convert_to_dict(self, query_results):
+        res = {}
+
+        for query_result in query_results:
+            if query_result.injury_severity not in res:
+                res[query_result.injury_severity] = {}
+            if query_result.accident_year not in res[query_result.injury_severity]:
+                res[query_result.injury_severity][query_result.accident_year] = 0
+
+            res[query_result.injury_severity][query_result.accident_year] += query_result.count
+
+        return res
+
     def __init__(self, request_params: RequestParams):
         super().__init__(request_params, type(self).name)
-        self.rank = 18
-        self.text = {"title": "נפגעים הולכי רגל ברחוב ז׳בוטינסקי, פתח תקווה"}
 
-    @staticmethod
-    def is_in_cache() -> bool:
-        return False
+        self.rank = 18
+        self.text = {
+            "title": f"נפגעים הולכי רגל ב- {get_news_flash_location_text(request_params.news_flash_obj)}",
+            "labels": gen_entity_labels(InjurySeverity)
```

Comment:
Ziv had a bug in localization, please add labels in localize_items function ([See this pr](https://github.com/hasadna/anyway/pull/1915/files))